.npmrc

```diff
@@ -2,3 +2,4 @@ auto-install-peers=true
 public-hoist-pattern[]='*eslint*'
 public-hoist-pattern[]='*prettier*'
 public-hoist-pattern[]='@emotion/react'
+strict-peer-dependencies=false
```

Comment:
As you mentioned you'd like to get rid of this from `.npmrc`, you might find useful the `"pnpm": {` settings in package.json. I recently used it to work around dependency problems.

https://github.com/system-ui/theme-ui/blob/27ea4123d548d67891d3a2c1f811b41ea86398d4/package.json#L94-L110

Reply:
Oooh thanks for the tip! Even temporarily, felt like a terrible fix 🙈

Reply:
Just switched to this.